architectds/architectds.py

```diff
@@ -491,8 +491,8 @@ def generate_elf(self):
 
         includedirs = self.includedirs + [self.out_assets_path]
 
-        includeflags = ' '.join('-I' + path + '/include' for path in self.libdirs) + \
-                       ' ' + ' '.join('-I' + path for path in includedirs)
+        includeflags = ' '.join('-isystem ' + path + '/include' for path in self.libdirs) + \
+                       ' ' + ' '.join('-isystem ' + path for path in includedirs)
```

Comment:
This is okay for includes that come from libraries (`self.libdirs + '/include'`), but not okay for every other include directory in `includedirs`. Can you leave that other change out? In both functions.

Reply:
ah, you're totally right. Yeah, lemme fix that!